install/operator/pkg/syndesis/configuration/configuration.go

```diff
@@ -353,12 +354,12 @@ func GetProperties(ctx context.Context, file string, clientTools *clienttools.Cl
 		if err != nil {
 			return nil, err
 		}
-	}
 
-	if ac, err := capabilities.ApiCapabilities(clientTools); err != nil {
-		return nil, err
-	} else {
-		configuration.ApiServer = *ac
+		if ac, err := capabilities.ApiCapabilities(clientTools); err != nil {
+			return nil, err
+		} else {
+			configuration.ApiServer = *ac
+		}
```

Comment:
Is there something causing problems with this?

Reply:
Yes. As commented above I don't need a client tool for `olm` command. this command will run in CI systems most likely. The block of code I put inside the condition throws an error if the clienttool is nil.